src/cleanvision/utils/viz_manager.py

```diff
@@ -45,6 +45,12 @@ def plot_image_grid(
     fig, axes = plt.subplots(
         nrows, ncols, figsize=(cell_size[0] * ncols, cell_size[1] * nrows)
     )
+
+    for i in range(len(images)):
+        title_width = len(titles[i]) * 0.09722
```

Comment:
please comment code to describe this constant and how it was chosen

Reply:
it should also be defined as a variable like `CHARACTER_SIZE`, and then you should reference this variable below instead of repeating the constant. This makes it easier to change the value of the constant in the future

Reply:
Sure! I felt this would be helpful for future developers.